core/win32/loader.c

```diff
@@ -1239,7 +1239,7 @@ privload_process_one_import(privmod_t *mod, privmod_t *impmod,
          */
         if (forwfunc == (char *)(ptr_int_t)1 || strchr(forwfunc+1, '.') != NULL) {
             CLIENT_ASSERT(false, "unexpected forwarder string");
-            return NULL;
```

Comment:
Is this change and similar changes further down related to the  ´'variable' hides previous declaration´ warning?